src/App.tsx

```diff
@@ -1,57 +1,27 @@
-import React from 'react';
+import { useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Autocomplete } from './components/Autocomplete';
```

Comment:
add this file to git to make review possible

P.S. Don't use `git commit -am 'your message example'` command - it doesn't add new files
better use `git add .` to add all files (or for example `git add src/components/Autocomplete.tsx` to add 1 file you need) and then just `git commit -m 'message'`